airbyte-cdk/python/airbyte_cdk/destinations/vector_db_based/batcher.py

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+from typing import Any, Callable, List
+
+
+class Batcher:
```

Comment:
Can we add class-level docstrings for each class? To address:
- What is the class responsible for.
- How should it iteract with other classes.
- When it is expected to be subclassed by developers (or not).